ui/src/components/homepage/Easter.js

```diff
@@ -0,0 +1,160 @@
+import React from "react";
+import {
+    Flex, 
+    Button, 
+    Text,
+    Stack,
+    Link,
+    SimpleGrid
+  } from '@chakra-ui/react';
+import { AiOutlineDesktop, AiOutlineInstagram } from 'react-icons/ai';
+import { GiPartyPopper } from 'react-icons/gi';
+import { useMediaQuery } from '@chakra-ui/media-query';
+
+const HeroSection = () => {
```

Comment:
hmm, it's a bit confusing to name this `HeroSection` and have it different from the file name and also import under a different name in MainContainer.js